<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Results (powered by Planemo)"
           tests="27"
           errors="1"
           failures="4"
           skip="0">
    
    <testcase classname="toolshed.g2.bx.psu.edu/repos/bgruening/flye/flye (Test #1)" name="0" time="236.13551425933838">
        
            
            <system-out><![CDATA[
            [2023-01-13 04:51:01] INFO: Starting Flye 2.6-release[2023-01-13 04:51:01] INFO: >>>STAGE: configure[2023-01-13 04:51:01] INFO: Configuring run[2023-01-13 04:51:01] INFO: Total read length: 1466260[2023-01-13 04:51:01] INFO: Input genome size: 10000[2023-01-13 04:51:01] INFO: Estimated coverage: 146[2023-01-13 04:51:01] INFO: Reads N50/N90: 9054 / 4697[2023-01-13 04:51:01] INFO: Minimum overlap set to 5000[2023-01-13 04:51:01] INFO: Selected k-mer size: 15[2023-01-13 04:51:01] INFO: >>>STAGE: assembly[2023-01-13 04:51:01] INFO: Assembling disjointigs[2023-01-13 04:51:01] INFO: Reading sequences[2023-01-13 04:51:01] INFO: Generating solid k-mer index[2023-01-13 04:51:21] INFO: Counting k-mers (1/2):0% 10% 20% 30% 40% 50% 60% 70% 80% 90% 100% [2023-01-13 04:51:21] INFO: Counting k-mers (2/2):0% 10% 20% 30% 40% 50% 60% 70% 80% 90% 100% [2023-01-13 04:51:22] INFO: Filling index table0% 10% 20% 30% 40% 50% 60% 70% 80% 90% 100% [2023-01-13 04:52:02] INFO: Extending reads[2023-01-13 04:52:07] INFO: Overlap-based coverage: 25[2023-01-13 04:52:07] INFO: Median overlap divergence: 0.1692240% 10% 20% 30% 40% 50% 60% 70% 80% 90% 100% [2023-01-13 04:52:09] INFO: Assembled 1 disjointigs[2023-01-13 04:52:09] INFO: Generating sequence10% 20% 40% 50% 70% 80% 100% [2023-01-13 04:52:10] INFO: >>>STAGE: consensus[2023-01-13 04:52:10] INFO: Running Minimap2[2023-01-13 04:52:11] INFO: Computing consensus[2023-01-13 04:52:13] INFO: Alignment error rate: 0.156879[2023-01-13 04:52:13] INFO: >>>STAGE: repeat[2023-01-13 04:52:13] INFO: Building and resolving repeat graph[2023-01-13 04:52:13] INFO: Reading sequences[2023-01-13 04:52:13] INFO: Building repeat graph50% 100% [2023-01-13 04:52:33] INFO: Median overlap divergence: 0[2023-01-13 04:52:33] INFO: Aligning reads to the graph0% 10% 20% 30% 40% 50% 60% 70% 80% 90% 100% [2023-01-13 04:52:54] INFO: Aligned read sequence: 1184744 / 1313105 (0.902246)[2023-01-13 04:52:54] INFO: Median overlap divergence: 0.0616647[2023-01-13 04:52:54] INFO: Mean edge coverage: 29[2023-01-13 04:52:54] INFO: Resolving repeats[2023-01-13 04:52:54] INFO: >>>STAGE: trestle[2023-01-13 04:52:54] INFO: Simple unbridged repeats: 0[2023-01-13 04:52:54] INFO: Resolved: 0[2023-01-13 04:52:54] INFO: >>>STAGE: contigger[2023-01-13 04:52:54] INFO: Generating contigs[2023-01-13 04:52:54] INFO: Reading sequences[2023-01-13 04:52:54] INFO: Generated 1 contigs[2023-01-13 04:52:54] INFO: Added 0 scaffold connections[2023-01-13 04:52:54] INFO: >>>STAGE: polishing[2023-01-13 04:52:54] INFO: Polishing genome (1/1)[2023-01-13 04:52:54] INFO: Running minimap2Traceback (most recent call last):  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap    self.run()  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 99, in run    self._target(*self._args, **self._kwargs)  File "/usr/local/lib/python3.7/multiprocessing/managers.py", line 577, in _run_server    server.serve_forever()  File "/usr/local/lib/python3.7/multiprocessing/managers.py", line 173, in serve_forever    sys.exit(0)SystemExit: 0During handling of the above exception, another exception occurred:Traceback (most recent call last):  File "/usr/local/lib/python3.7/multiprocessing/util.py", line 265, in _run_finalizers    finalizer()  File "/usr/local/lib/python3.7/multiprocessing/util.py", line 189, in __call__    res = self._callback(*self._args, **self._kwargs)  File "/usr/local/lib/python3.7/shutil.py", line 491, in rmtree    _rmtree_safe_fd(fd, path, onerror)  File "/usr/local/lib/python3.7/shutil.py", line 449, in _rmtree_safe_fd    onerror(os.unlink, fullname, sys.exc_info())  File "/usr/local/lib/python3.7/shutil.py", line 447, in _rmtree_safe_fd    os.unlink(entry.name, dir_fd=topfd)OSError: [Errno 16] Device or resource busy: '.nfs000000000044407c00000029'[2023-01-13 04:52:55] INFO: Separating alignment into bubbles[2023-01-13 04:52:57] INFO: Alignment error rate: 0.104507[2023-01-13 04:52:57] INFO: Correcting bubbles0% 10% 20% 30% 40% 50% 60% 70% 80% 90% 100% Traceback (most recent call last):  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap    self.run()  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 99, in run    self._target(*self._args, **self._kwargs)  File "/usr/local/lib/python3.7/multiprocessing/managers.py", line 577, in _run_server    server.serve_forever()  File "/usr/local/lib/python3.7/multiprocessing/managers.py", line 173, in serve_forever    sys.exit(0)SystemExit: 0During handling of the above exception, another exception occurred:Traceback (most recent call last):  File "/usr/local/lib/python3.7/multiprocessing/util.py", line 265, in _run_finalizers    finalizer()  File "/usr/local/lib/python3.7/multiprocessing/util.py", line 189, in __call__    res = self._callback(*self._args, **self._kwargs)  File "/usr/local/lib/python3.7/shutil.py", line 491, in rmtree    _rmtree_safe_fd(fd, path, onerror)  File "/usr/local/lib/python3.7/shutil.py", line 449, in _rmtree_safe_fd    onerror(os.unlink, fullname, sys.exc_info())  File "/usr/local/lib/python3.7/shutil.py", line 447, in _rmtree_safe_fd    os.unlink(entry.name, dir_fd=topfd)OSError: [Errno 16] Device or resource busy: '.nfs00000000004440970000002a'[2023-01-13 04:53:03] INFO: >>>STAGE: finalize[2023-01-13 04:53:03] INFO: Assembly statistics:Total length:39944Fragments:1Fragments N50:39944Largest frg:39944Scaffolds:0Mean coverage:27[2023-01-13 04:53:03] INFO: Final assembly: /galaxy/server/database/jobs_directory/000/38/working/out_dir/assembly.fasta
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="toolshed.g2.bx.psu.edu/repos/bgruening/flye/flye (Test #2)" name="1" time="247.21047139167786">
        
            
            <system-out><![CDATA[
            [2023-01-13 04:51:55] INFO: Starting Flye 2.6-release[2023-01-13 04:51:55] INFO: >>>STAGE: configure[2023-01-13 04:51:55] INFO: Configuring run[2023-01-13 04:51:55] INFO: Total read length: 1466260[2023-01-13 04:51:55] INFO: Input genome size: 10000[2023-01-13 04:51:55] INFO: Estimated coverage: 146[2023-01-13 04:51:55] INFO: Reads N50/N90: 9054 / 4697[2023-01-13 04:51:55] INFO: Minimum overlap set to 5000[2023-01-13 04:51:55] INFO: Selected k-mer size: 15[2023-01-13 04:51:55] INFO: >>>STAGE: assembly[2023-01-13 04:51:55] INFO: Assembling disjointigs[2023-01-13 04:51:55] INFO: Reading sequences[2023-01-13 04:51:55] INFO: Generating solid k-mer index[2023-01-13 04:52:15] INFO: Counting k-mers (1/2):0% 10% 20% 30% 40% 50% 60% 70% 80% 90% 100% [2023-01-13 04:52:16] INFO: Counting k-mers (2/2):0% 10% 20% 30% 40% 50% 60% 70% 80% 90% 100% [2023-01-13 04:52:16] INFO: Filling index table0% 10% 20% 30% 40% 50% 60% 70% 80% 90% 100% [2023-01-13 04:52:57] INFO: Extending reads[2023-01-13 04:53:03] INFO: Overlap-based coverage: 25[2023-01-13 04:53:03] INFO: Median overlap divergence: 0.1692240% 10% 20% 30% 40% 50% 60% 70% 80% 90% 100% [2023-01-13 04:53:04] INFO: Assembled 1 disjointigs[2023-01-13 04:53:04] INFO: Generating sequence10% 20% 40% 50% 70% 80% 100% [2023-01-13 04:53:05] INFO: >>>STAGE: consensus[2023-01-13 04:53:05] INFO: Running Minimap2[2023-01-13 04:53:06] INFO: Computing consensus[2023-01-13 04:53:08] INFO: Alignment error rate: 0.157030[2023-01-13 04:53:08] INFO: >>>STAGE: repeat[2023-01-13 04:53:08] INFO: Building and resolving repeat graph[2023-01-13 04:53:08] INFO: Reading sequences[2023-01-13 04:53:08] INFO: Building repeat graph50% 100% [2023-01-13 04:53:28] INFO: Median overlap divergence: 0[2023-01-13 04:53:28] INFO: Aligning reads to the graph0% 10% 20% 30% 40% 50% 60% 70% 80% 90% 100% [2023-01-13 04:53:49] INFO: Aligned read sequence: 1184737 / 1313105 (0.902241)[2023-01-13 04:53:49] INFO: Median overlap divergence: 0.0613459[2023-01-13 04:53:49] INFO: Mean edge coverage: 29[2023-01-13 04:53:49] INFO: Resolving repeats[2023-01-13 04:53:49] INFO: >>>STAGE: trestle[2023-01-13 04:53:49] INFO: Simple unbridged repeats: 0[2023-01-13 04:53:49] INFO: Resolved: 0[2023-01-13 04:53:49] INFO: >>>STAGE: contigger[2023-01-13 04:53:49] INFO: Generating contigs[2023-01-13 04:53:49] INFO: Reading sequences[2023-01-13 04:53:49] INFO: Generated 1 contigs[2023-01-13 04:53:49] INFO: Added 0 scaffold connections[2023-01-13 04:53:49] INFO: >>>STAGE: polishing[2023-01-13 04:53:49] INFO: Polishing genome (1/1)[2023-01-13 04:53:49] INFO: Running minimap2Traceback (most recent call last):  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap    self.run()  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 99, in run    self._target(*self._args, **self._kwargs)  File "/usr/local/lib/python3.7/multiprocessing/managers.py", line 577, in _run_server    server.serve_forever()  File "/usr/local/lib/python3.7/multiprocessing/managers.py", line 173, in serve_forever    sys.exit(0)SystemExit: 0During handling of the above exception, another exception occurred:Traceback (most recent call last):  File "/usr/local/lib/python3.7/multiprocessing/util.py", line 265, in _run_finalizers    finalizer()  File "/usr/local/lib/python3.7/multiprocessing/util.py", line 189, in __call__    res = self._callback(*self._args, **self._kwargs)  File "/usr/local/lib/python3.7/shutil.py", line 491, in rmtree    _rmtree_safe_fd(fd, path, onerror)  File "/usr/local/lib/python3.7/shutil.py", line 449, in _rmtree_safe_fd    onerror(os.unlink, fullname, sys.exc_info())  File "/usr/local/lib/python3.7/shutil.py", line 447, in _rmtree_safe_fd    os.unlink(entry.name, dir_fd=topfd)OSError: [Errno 16] Device or resource busy: '.nfs00000000004440b800000031'[2023-01-13 04:53:50] INFO: Separating alignment into bubbles[2023-01-13 04:53:52] INFO: Alignment error rate: 0.104939[2023-01-13 04:53:52] INFO: Correcting bubbles0% 10% 20% 30% 40% 50% 60% 70% 80% 90% 100% Traceback (most recent call last):  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap    self.run()  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 99, in run    self._target(*self._args, **self._kwargs)  File "/usr/local/lib/python3.7/multiprocessing/managers.py", line 577, in _run_server    server.serve_forever()  File "/usr/local/lib/python3.7/multiprocessing/managers.py", line 173, in serve_forever    sys.exit(0)SystemExit: 0During handling of the above exception, another exception occurred:Traceback (most recent call last):  File "/usr/local/lib/python3.7/multiprocessing/util.py", line 265, in _run_finalizers    finalizer()  File "/usr/local/lib/python3.7/multiprocessing/util.py", line 189, in __call__    res = self._callback(*self._args, **self._kwargs)  File "/usr/local/lib/python3.7/shutil.py", line 491, in rmtree    _rmtree_safe_fd(fd, path, onerror)  File "/usr/local/lib/python3.7/shutil.py", line 449, in _rmtree_safe_fd    onerror(os.unlink, fullname, sys.exc_info())  File "/usr/local/lib/python3.7/shutil.py", line 447, in _rmtree_safe_fd    os.unlink(entry.name, dir_fd=topfd)OSError: [Errno 16] Device or resource busy: '.nfs00000000004441270000003a'[2023-01-13 04:53:57] INFO: >>>STAGE: finalize[2023-01-13 04:53:57] INFO: Assembly statistics:Total length:40419Fragments:1Fragments N50:40419Largest frg:40419Scaffolds:0Mean coverage:26[2023-01-13 04:53:57] INFO: Final assembly: /galaxy/server/database/jobs_directory/000/42/working/out_dir/assembly.fasta
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="toolshed.g2.bx.psu.edu/repos/bgruening/flye/flye (Test #3)" name="2" time="240.3693492412567">
        
            
            <system-out><![CDATA[
            [2023-01-13 04:51:58] INFO: Starting Flye 2.6-release[2023-01-13 04:51:58] INFO: >>>STAGE: configure[2023-01-13 04:51:58] INFO: Configuring run[2023-01-13 04:51:58] INFO: Total read length: 1466260[2023-01-13 04:51:58] INFO: Input genome size: 10000[2023-01-13 04:51:58] INFO: Estimated coverage: 146[2023-01-13 04:51:58] INFO: Reads N50/N90: 9054 / 4697[2023-01-13 04:51:58] INFO: Minimum overlap set to 5000[2023-01-13 04:51:58] INFO: Selected k-mer size: 17[2023-01-13 04:51:58] INFO: Using longest 40x reads for contig assembly[2023-01-13 04:51:58] INFO: >>>STAGE: assembly[2023-01-13 04:51:58] INFO: Assembling disjointigs[2023-01-13 04:51:58] INFO: Reading sequences[2023-01-13 04:51:58] INFO: Generating solid k-mer index[2023-01-13 04:52:18] INFO: Counting k-mers (1/2):0% 10% 20% 30% 40% 50% 60% 70% 80% 90% 100% [2023-01-13 04:52:18] INFO: Counting k-mers (2/2):0% 10% 20% 30% 40% 50% 60% 70% 80% 90% 100% [2023-01-13 04:52:18] INFO: Filling index table0% 10% 20% 30% 40% 50% 60% 70% 80% 90% 100% [2023-01-13 04:52:53] INFO: Extending reads[2023-01-13 04:52:53] INFO: Overlap-based coverage: 3[2023-01-13 04:52:53] INFO: Median overlap divergence: 0.1735210% 10% 20% 30% 40% 50% 60% 70% 80% 90% 100% [2023-01-13 04:52:53] INFO: Assembled 1 disjointigs[2023-01-13 04:52:53] INFO: Generating sequence10% 20% 40% 50% 70% 80% 100% [2023-01-13 04:52:55] INFO: >>>STAGE: consensus[2023-01-13 04:52:55] INFO: Running Minimap2[2023-01-13 04:52:56] INFO: Computing consensus[2023-01-13 04:52:57] INFO: Alignment error rate: 0.146313[2023-01-13 04:52:57] INFO: >>>STAGE: repeat[2023-01-13 04:52:57] INFO: Building and resolving repeat graph[2023-01-13 04:52:57] INFO: Reading sequences[2023-01-13 04:52:58] INFO: Building repeat graph50% 100% [2023-01-13 04:53:18] INFO: Median overlap divergence: 0[2023-01-13 04:53:18] INFO: Aligning reads to the graph0% 10% 20% 30% 40% 50% 60% 70% 80% 90% 100% [2023-01-13 04:53:39] INFO: Aligned read sequence: 1174263 / 1313105 (0.894264)[2023-01-13 04:53:39] INFO: Median overlap divergence: 0.0589391[2023-01-13 04:53:39] INFO: Mean edge coverage: 29[2023-01-13 04:53:39] INFO: Resolving repeats[2023-01-13 04:53:39] INFO: >>>STAGE: contigger[2023-01-13 04:53:39] INFO: Generating contigs[2023-01-13 04:53:39] INFO: Reading sequences[2023-01-13 04:53:39] INFO: Generated 1 contigs[2023-01-13 04:53:39] INFO: Added 0 scaffold connections[2023-01-13 04:53:39] INFO: >>>STAGE: polishing[2023-01-13 04:53:39] INFO: Polishing genome (1/2)[2023-01-13 04:53:39] INFO: Running minimap2[2023-01-13 04:53:40] INFO: Separating alignment into bubblesTraceback (most recent call last):  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap    self.run()  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 99, in run    self._target(*self._args, **self._kwargs)  File "/usr/local/lib/python3.7/multiprocessing/managers.py", line 577, in _run_server    server.serve_forever()  File "/usr/local/lib/python3.7/multiprocessing/managers.py", line 173, in serve_forever    sys.exit(0)SystemExit: 0During handling of the above exception, another exception occurred:Traceback (most recent call last):  File "/usr/local/lib/python3.7/multiprocessing/util.py", line 265, in _run_finalizers    finalizer()  File "/usr/local/lib/python3.7/multiprocessing/util.py", line 189, in __call__    res = self._callback(*self._args, **self._kwargs)  File "/usr/local/lib/python3.7/shutil.py", line 491, in rmtree    _rmtree_safe_fd(fd, path, onerror)  File "/usr/local/lib/python3.7/shutil.py", line 449, in _rmtree_safe_fd    onerror(os.unlink, fullname, sys.exc_info())  File "/usr/local/lib/python3.7/shutil.py", line 447, in _rmtree_safe_fd    os.unlink(entry.name, dir_fd=topfd)OSError: [Errno 16] Device or resource busy: '.nfs000000000044409a0000002e'[2023-01-13 04:53:41] INFO: Alignment error rate: 0.099017[2023-01-13 04:53:41] INFO: Correcting bubbles0% 10% 20% 30% 40% 50% 60% 70% 80% 90% 100% Traceback (most recent call last):  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap    self.run()  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 99, in run    self._target(*self._args, **self._kwargs)  File "/usr/local/lib/python3.7/multiprocessing/managers.py", line 577, in _run_server    server.serve_forever()  File "/usr/local/lib/python3.7/multiprocessing/managers.py", line 173, in serve_forever    sys.exit(0)SystemExit: 0During handling of the above exception, another exception occurred:Traceback (most recent call last):  File "/usr/local/lib/python3.7/multiprocessing/util.py", line 265, in _run_finalizers    finalizer()  File "/usr/local/lib/python3.7/multiprocessing/util.py", line 189, in __call__    res = self._callback(*self._args, **self._kwargs)  File "/usr/local/lib/python3.7/shutil.py", line 491, in rmtree    _rmtree_safe_fd(fd, path, onerror)  File "/usr/local/lib/python3.7/shutil.py", line 449, in _rmtree_safe_fd    onerror(os.unlink, fullname, sys.exc_info())  File "/usr/local/lib/python3.7/shutil.py", line 447, in _rmtree_safe_fd    os.unlink(entry.name, dir_fd=topfd)OSError: [Errno 16] Device or resource busy: '.nfs00000000004440eb00000030'[2023-01-13 04:53:46] INFO: Polishing genome (2/2)[2023-01-13 04:53:46] INFO: Running minimap2[2023-01-13 04:53:47] INFO: Separating alignment into bubbles[2023-01-13 04:53:48] INFO: Alignment error rate: 0.094400[2023-01-13 04:53:48] INFO: Correcting bubbles0% 10% 20% 30% 40% 50% 60% 70% 80% 90% 100% [2023-01-13 04:53:53] INFO: >>>STAGE: finalize[2023-01-13 04:53:53] INFO: Assembly statistics:Total length:40414Fragments:1Fragments N50:40414Largest frg:40414Scaffolds:0Mean coverage:27[2023-01-13 04:53:53] INFO: Final assembly: /galaxy/server/database/jobs_directory/000/43/working/out_dir/assembly.fastaTraceback (most recent call last):  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap    self.run()  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 99, in run    self._target(*self._args, **self._kwargs)  File "/usr/local/lib/python3.7/multiprocessing/managers.py", line 577, in _run_server    server.serve_forever()  File "/usr/local/lib/python3.7/multiprocessing/managers.py", line 173, in serve_forever    sys.exit(0)SystemExit: 0During handling of the above exception, another exception occurred:Traceback (most recent call last):  File "/usr/local/lib/python3.7/multiprocessing/util.py", line 265, in _run_finalizers    finalizer()  File "/usr/local/lib/python3.7/multiprocessing/util.py", line 189, in __call__    res = self._callback(*self._args, **self._kwargs)  File "/usr/local/lib/python3.7/shutil.py", line 491, in rmtree    _rmtree_safe_fd(fd, path, onerror)  File "/usr/local/lib/python3.7/shutil.py", line 449, in _rmtree_safe_fd    onerror(os.unlink, fullname, sys.exc_info())  File "/usr/local/lib/python3.7/shutil.py", line 447, in _rmtree_safe_fd    os.unlink(entry.name, dir_fd=topfd)OSError: [Errno 16] Device or resource busy: '.nfs00000000004440ec00000033'
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="toolshed.g2.bx.psu.edu/repos/bgruening/flye/flye (Test #4)" name="3" time="242.071106672287">
        
            
            <system-out><![CDATA[
            [2023-01-13 04:51:51] INFO: Starting Flye 2.6-release[2023-01-13 04:51:51] INFO: >>>STAGE: configure[2023-01-13 04:51:51] INFO: Configuring run[2023-01-13 04:51:51] INFO: Total read length: 1466260[2023-01-13 04:51:51] INFO: Input genome size: 10000[2023-01-13 04:51:51] INFO: Estimated coverage: 146[2023-01-13 04:51:51] INFO: Reads N50/N90: 9054 / 4697[2023-01-13 04:51:51] INFO: Minimum overlap set to 5000[2023-01-13 04:51:51] INFO: Selected k-mer size: 15[2023-01-13 04:51:51] INFO: >>>STAGE: assembly[2023-01-13 04:51:51] INFO: Assembling disjointigs[2023-01-13 04:51:51] INFO: Reading sequences[2023-01-13 04:51:52] INFO: Generating solid k-mer index[2023-01-13 04:52:12] INFO: Counting k-mers (1/2):0% 10% 20% 30% 40% 50% 60% 70% 80% 90% 100% [2023-01-13 04:52:12] INFO: Counting k-mers (2/2):0% 10% 20% 30% 40% 50% 60% 70% 80% 90% 100% [2023-01-13 04:52:12] INFO: Filling index table (1/2)0% 10% 20% 30% 40% 50% 60% 70% 80% 90% 100% [2023-01-13 04:52:13] INFO: Filling index table (2/2)0% 10% 20% 30% 40% 50% 60% 70% 80% 90% 100% [2023-01-13 04:52:52] INFO: Extending reads[2023-01-13 04:52:57] INFO: Overlap-based coverage: 24[2023-01-13 04:52:57] INFO: Median overlap divergence: 0.1841870% 10% 20% 30% 40% 50% 60% 70% 80% 90% 100% [2023-01-13 04:52:58] INFO: Assembled 1 disjointigs[2023-01-13 04:52:58] INFO: Generating sequence10% 20% 30% 40% 50% 60% 70% 80% 100% [2023-01-13 04:53:00] INFO: >>>STAGE: consensus[2023-01-13 04:53:00] INFO: Running Minimap2[2023-01-13 04:53:01] INFO: Computing consensus[2023-01-13 04:53:03] INFO: Alignment error rate: 0.183209[2023-01-13 04:53:03] INFO: >>>STAGE: repeat[2023-01-13 04:53:03] INFO: Building and resolving repeat graph[2023-01-13 04:53:03] INFO: Reading sequences[2023-01-13 04:53:03] INFO: Building repeat graph50% 100% [2023-01-13 04:53:23] INFO: Median overlap divergence: 0[2023-01-13 04:53:23] INFO: Aligning reads to the graph0% 10% 20% 30% 40% 50% 60% 70% 80% 90% 100% [2023-01-13 04:53:44] INFO: Aligned read sequence: 1158698 / 1313105 (0.882411)[2023-01-13 04:53:44] INFO: Median overlap divergence: 0.0642069[2023-01-13 04:53:44] INFO: Mean edge coverage: 32[2023-01-13 04:53:44] INFO: Resolving repeats[2023-01-13 04:53:44] INFO: >>>STAGE: plasmids[2023-01-13 04:53:44] INFO: Recovering short unassembled sequencesTraceback (most recent call last):  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap    self.run()  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 99, in run    self._target(*self._args, **self._kwargs)  File "/usr/local/lib/python3.7/multiprocessing/managers.py", line 577, in _run_server    server.serve_forever()  File "/usr/local/lib/python3.7/multiprocessing/managers.py", line 173, in serve_forever    sys.exit(0)SystemExit: 0During handling of the above exception, another exception occurred:Traceback (most recent call last):  File "/usr/local/lib/python3.7/multiprocessing/util.py", line 265, in _run_finalizers    finalizer()  File "/usr/local/lib/python3.7/multiprocessing/util.py", line 189, in __call__    res = self._callback(*self._args, **self._kwargs)  File "/usr/local/lib/python3.7/shutil.py", line 491, in rmtree    _rmtree_safe_fd(fd, path, onerror)  File "/usr/local/lib/python3.7/shutil.py", line 449, in _rmtree_safe_fd    onerror(os.unlink, fullname, sys.exc_info())  File "/usr/local/lib/python3.7/shutil.py", line 447, in _rmtree_safe_fd    os.unlink(entry.name, dir_fd=topfd)OSError: [Errno 16] Device or resource busy: '.nfs00000000004440a10000002f'[2023-01-13 04:53:45] INFO: Added 0 extra contigs[2023-01-13 04:53:45] INFO: >>>STAGE: contigger[2023-01-13 04:53:45] INFO: Generating contigs[2023-01-13 04:53:45] INFO: Reading sequences[2023-01-13 04:53:45] INFO: Generated 1 contigs[2023-01-13 04:53:45] INFO: Added 0 scaffold connections[2023-01-13 04:53:45] INFO: >>>STAGE: polishing[2023-01-13 04:53:45] INFO: Polishing genome (1/1)[2023-01-13 04:53:45] INFO: Running minimap2[2023-01-13 04:53:46] INFO: Separating alignment into bubbles[2023-01-13 04:53:47] INFO: Alignment error rate: 0.115105[2023-01-13 04:53:47] INFO: Correcting bubbles0% 10% 20% 30% 40% 50% 60% 70% 80% 90% 100% Traceback (most recent call last):  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap    self.run()  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 99, in run    self._target(*self._args, **self._kwargs)  File "/usr/local/lib/python3.7/multiprocessing/managers.py", line 577, in _run_server    server.serve_forever()  File "/usr/local/lib/python3.7/multiprocessing/managers.py", line 173, in serve_forever    sys.exit(0)SystemExit: 0During handling of the above exception, another exception occurred:Traceback (most recent call last):  File "/usr/local/lib/python3.7/multiprocessing/util.py", line 265, in _run_finalizers    finalizer()  File "/usr/local/lib/python3.7/multiprocessing/util.py", line 189, in __call__    res = self._callback(*self._args, **self._kwargs)  File "/usr/local/lib/python3.7/shutil.py", line 491, in rmtree    _rmtree_safe_fd(fd, path, onerror)  File "/usr/local/lib/python3.7/shutil.py", line 449, in _rmtree_safe_fd    onerror(os.unlink, fullname, sys.exc_info())  File "/usr/local/lib/python3.7/shutil.py", line 447, in _rmtree_safe_fd    os.unlink(entry.name, dir_fd=topfd)OSError: [Errno 16] Device or resource busy: '.nfs00000000004440fe00000036'Traceback (most recent call last):  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap    self.run()  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 99, in run    self._target(*self._args, **self._kwargs)  File "/usr/local/lib/python3.7/multiprocessing/managers.py", line 577, in _run_server    server.serve_forever()  File "/usr/local/lib/python3.7/multiprocessing/managers.py", line 173, in serve_forever    sys.exit(0)SystemExit: 0During handling of the above exception, another exception occurred:Traceback (most recent call last):  File "/usr/local/lib/python3.7/multiprocessing/util.py", line 265, in _run_finalizers    finalizer()  File "/usr/local/lib/python3.7/multiprocessing/util.py", line 189, in __call__    res = self._callback(*self._args, **self._kwargs)  File "/usr/local/lib/python3.7/shutil.py", line 491, in rmtree    _rmtree_safe_fd(fd, path, onerror)  File "/usr/local/lib/python3.7/shutil.py", line 449, in _rmtree_safe_fd    onerror(os.unlink, fullname, sys.exc_info())  File "/usr/local/lib/python3.7/shutil.py", line 447, in _rmtree_safe_fd    os.unlink(entry.name, dir_fd=topfd)OSError: [Errno 16] Device or resource busy: '.nfs000000000044410a00000037'[2023-01-13 04:53:54] INFO: >>>STAGE: finalize[2023-01-13 04:53:54] INFO: Assembly statistics:Total length:35685Fragments:1Fragments N50:35685Largest frg:35685Scaffolds:0Mean coverage:36[2023-01-13 04:53:54] INFO: Final assembly: /galaxy/server/database/jobs_directory/000/44/working/out_dir/assembly.fasta
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="toolshed.g2.bx.psu.edu/repos/devteam/vcfdistance/vcfdistance (Test #1)" name="0" time="115.61206674575806">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="toolshed.g2.bx.psu.edu/repos/devteam/vcffilter/vcffilter2 (Test #1)" name="0" time="147.25412487983704">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="toolshed.g2.bx.psu.edu/repos/devteam/vcffilter/vcffilter2 (Test #2)" name="1" time="145.2012062072754">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="toolshed.g2.bx.psu.edu/repos/devteam/vcfselectsamples/vcfselectsamples (Test #1)" name="0" time="161.80679059028625">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="toolshed.g2.bx.psu.edu/repos/devteam/vcfsort/vcfsort (Test #1)" name="0" time="50.708184003829956">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="toolshed.g2.bx.psu.edu/repos/devteam/vcftools_annotate/vcftools_annotate (Test #1)" name="0" time="117.28618025779724">
        
            
                <error type="error" message="Tool exit code: 127"><![CDATA[
                    { "inputs": {  "condition": "\u003e= 7",  "description": "MinAF [7]",  "filter": "MinAF",  "input": {   "id": "5376787bc5572a09",   "src": "hda"  },  "tag": "FORMAT/FREQ" }, "job": {  "command_line": "echo \"{ tag  =\u003e \u0027FORMAT/FREQ\u0027, name =\u003e \u0027MinAF\u0027, desc =\u003e \u0027MinAF [7]\u0027, test =\u003e sub { my @t = split(\u0027%\u0027, @\\$MATCH[0]); return @t[0] \u003e= 7 ? \\$PASS : \\$FAIL }, },\" \u003e f.txt ;  vcf-annotate -f f.txt /galaxy/server/database/objects/c/e/9/dataset_ce9fac87-0de0-4dbc-b231-f25954725fa0.dat \u003e /galaxy/server/database/objects/d/0/0/dataset_d00ff76b-092c-49f0-8c6f-d2ee3419fc8a.dat",  "command_version": "",  "create_time": "2023-01-13T04:45:12.582835",  "dependencies": [],  "exit_code": 127,  "external_id": "gxy-jg558",  "galaxy_version": "21.09",  "history_id": "702fa106437bef85",  "id": "47511bac4c7889dc",  "inputs": {   "input": {    "id": "5376787bc5572a09",    "src": "hda",    "uuid": "ce9fac87-0de0-4dbc-b231-f25954725fa0"   }  },  "job_messages": [   {    "code_desc": "tool progress",    "desc": "Log: tool progress",    "error_level": 1,    "match": "",    "stream": "stderr",    "type": "regex"   },   {    "code_desc": "tool progress",    "desc": "Log: tool progress",    "error_level": 1,    "match": "",    "stream": "stdout",    "type": "regex"   }  ],  "job_metrics": [   {    "name": "galaxy_slots",    "plugin": "core",    "raw_value": "1.0000000",    "title": "Cores Allocated",    "value": "1"   },   {    "name": "galaxy_memory_mb",    "plugin": "core",    "raw_value": "2000.0000000",    "title": "Memory Allocated (MB)",    "value": "2000"   },   {    "name": "start_epoch",    "plugin": "core",    "raw_value": "1673585197.0000000",    "title": "Job Start Time",    "value": "2023-01-13 04:46:37"   },   {    "name": "end_epoch",    "plugin": "core",    "raw_value": "1673585197.0000000",    "title": "Job End Time",    "value": "2023-01-13 04:46:37"   },   {    "name": "runtime_seconds",    "plugin": "core",    "raw_value": "0E-7",    "title": "Job Runtime (Wall Clock)",    "value": "0 seconds"   }  ],  "job_stderr": "/galaxy/server/database/jobs_directory/000/22/tool_script.sh: line 22: vcf-annotate: command not found\n",  "job_stdout": "",  "model_class": "Job",  "output_collections": {},  "outputs": {   "output": {    "id": "47511bac4c7889dc",    "src": "hda",    "uuid": "d00ff76b-092c-49f0-8c6f-d2ee3419fc8a"   }  },  "params": {   "__input_ext": "\"vcf\"",   "chromInfo": "\"/cvmfs/data.galaxyproject.org/managed/len/ucsc/?.len\"",   "condition": "\"\u003e= 7\"",   "dbkey": "\"?\"",   "description": "\"MinAF [7]\"",   "filter": "\"MinAF\"",   "tag": "\"FORMAT/FREQ\""  },  "state": "ok",  "stderr": "\n/galaxy/server/database/jobs_directory/000/22/tool_script.sh: line 22: vcf-annotate: command not found\n",  "stdout": "",  "tool_id": "toolshed.g2.bx.psu.edu/repos/devteam/vcftools_annotate/vcftools_annotate/0.1",  "tool_stderr": "",  "tool_stdout": "",  "traceback": null,  "update_time": "2023-01-13T04:46:49.440495",  "user_email": "tests@fake.org" }, "output_problems": [  "Output output:  different than expected, difference (using diff):\n( /tmp/tmpnr04azi9test_out1.vcf v. /tmp/tmp605ytxh9test_out1.vcf )\n--- local_file\n+++ history_data\n@@ -1,30 +0,0 @@\n-##fileformat=VCFv4.1\n-##FILTER=\u003cID=MinAF,Description=\"MinAF [7]\"\u003e\n-##source_20130813.1=vcf-annotate(r797) -f f.txt \u003cinput_file\u003e\n-#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tSample1\n-chr1\t14470\t.\tG\tA\t.\tPASS\tADP=25;WT=0;HET=1;HOM=0;NC=0\tGT:GQ:SDP:DP:RD:AD:FREQ:PVAL:RBQ:ABQ:RDF:RDR:ADF:ADR\t0/1:0:28:25:23:2:8%:9.8E-1:33:39:22:1:2:0\n-chr1\t14513\t.\tG\tA\t.\tPASS\tADP=32;WT=0;HET=1;HOM=0;NC=0\tGT:GQ:SDP:DP:RD:AD:FREQ:PVAL:RBQ:ABQ:RDF:RDR:ADF:ADR\t0/1:0:36:32:24:8:25%:9.8E-1:31:22:16:8:7:1\n-chr1\t14626\t.\tG\tC\t.\tPASS\tADP=44;WT=0;HET=1;HOM=0;NC=0\tGT:GQ:SDP:DP:RD:AD:FREQ:PVAL:RBQ:ABQ:RDF:RDR:ADF:ADR\t0/1:0:45:44:40:4:9.09%:9.8E-1:34:36:14:26:0:4\n-chr1\t14677\t.\tG\tA\t.\tPASS\tADP=70;WT=0;HET=1;HOM=0;NC=0\tGT:GQ:SDP:DP:RD:AD:FREQ:PVAL:RBQ:ABQ:RDF:RDR:ADF:ADR\t0/1:0:82:70:63:7:10%:9.8E-1:33:17:44:19:7:0\n-chr1\t14813\t.\tT\tG\t.\tMinAF\tADP=105;WT=0;HET=1;HOM=0;NC=0\tGT:GQ:SDP:DP:RD:AD:FREQ:PVAL:RBQ:ABQ:RDF:RDR:ADF:ADR\t0/1:0:121:105:103:2:1.9%:9.8E-1:32:22:42:61:1:1\n-chr1\t14815\t.\tC\tT\t.\tMinAF\tADP=117;WT=0;HET=1;HOM=0;NC=0\tGT:GQ:SDP:DP:RD:AD:FREQ:PVAL:RBQ:ABQ:RDF:RDR:ADF:ADR\t0/1:0:122:117:109:8:6.84%:9.8E-1:33:29:50:59:4:4\n-chr1\t14907\t.\tA\tG\t.\tPASS\tADP=104;WT=0;HET=1;HOM=0;NC=0\tGT:GQ:SDP:DP:RD:AD:FREQ:PVAL:RBQ:ABQ:RDF:RDR:ADF:ADR\t0/1:0:108:104:19:16:45.71%:9.8E-1:37:18:10:9:8:8\n-chr1\t14930\t.\tA\tG\t.\tPASS\tADP=103;WT=0;HET=1;HOM=0;NC=0\tGT:GQ:SDP:DP:RD:AD:FREQ:PVAL:RBQ:ABQ:RDF:RDR:ADF:ADR\t0/1:0:110:103:17:17:50%:9.8E-1:37:24:8:9:8:9\n-chr1\t15015\t.\tG\tC\t.\tPASS\tADP=52;WT=0;HET=1;HOM=0;NC=0\tGT:GQ:SDP:DP:RD:AD:FREQ:PVAL:RBQ:ABQ:RDF:RDR:ADF:ADR\t0/1:0:55:52:44:8:15.38%:9.8E-1:34:35:5:39:1:7\n-chr1\t15211\t.\tT\tG\t.\tPASS\tADP=21;WT=0;HET=0;HOM=1;NC=0\tGT:GQ:SDP:DP:RD:AD:FREQ:PVAL:RBQ:ABQ:RDF:RDR:ADF:ADR\t1/1:0:23:21:0:2:100%:9.8E-1:0:26:0:0:1:1\n-chr1\t15527\t.\tG\tA\t.\tPASS\tADP=53;WT=0;HET=1;HOM=0;NC=0\tGT:GQ:SDP:DP:RD:AD:FREQ:PVAL:RBQ:ABQ:RDF:RDR:ADF:ADR\t0/1:0:57:53:29:5:14.71%:9.8E-1:32:23:19:10:3:2\n-chr1\t15688\t.\tC\tT\t.\tPASS\tADP=34;WT=0;HET=1;HOM=0;NC=0\tGT:GQ:SDP:DP:RD:AD:FREQ:PVAL:RBQ:ABQ:RDF:RDR:ADF:ADR\t0/1:0:36:34:11:4:26.67%:9.8E-1:35:26:6:5:1:3\n-chr1\t16257\t.\tG\tC\t.\tPASS\tADP=23;WT=0;HET=1;HOM=0;NC=0\tGT:GQ:SDP:DP:RD:AD:FREQ:PVAL:RBQ:ABQ:RDF:RDR:ADF:ADR\t0/1:0:26:23:20:2:9.09%:9.8E-1:35:17:7:13:0:2\n-chr1\t16288\t.\tC\tG\t.\tPASS\tADP=15;WT=0;HET=1;HOM=0;NC=0\tGT:GQ:SDP:DP:RD:AD:FREQ:PVAL:RBQ:ABQ:RDF:RDR:ADF:ADR\t0/1:0:16:15:11:3:21.43%:9.8E-1:34:36:3:8:1:2\n-chr1\t16298\t.\tC\tT\t.\tPASS\tADP=12;WT=0;HET=1;HOM=0;NC=0\tGT:GQ:SDP:DP:RD:AD:FREQ:PVAL:RBQ:ABQ:RDF:RDR:ADF:ADR\t0/1:0:12:12:9:2:18.18%:9.8E-1:34:30:2:7:1:1\n-chr1\t16378\t.\tT\tC\t.\tPASS\tADP=24;WT=0;HET=1;HOM=0;NC=0\tGT:GQ:SDP:DP:RD:AD:FREQ:PVAL:RBQ:ABQ:RDF:RDR:ADF:ADR\t0/1:0:24:24:11:12:52.17%:9.8E-1:37:34:7:4:8:4\n-chr1\t16495\t.\tG\tC\t.\tPASS\tADP=21;WT=0;HET=1;HOM=0;NC=0\tGT:GQ:SDP:DP:RD:AD:FREQ:PVAL:RBQ:ABQ:RDF:RDR:ADF:ADR\t0/1:0:22:21:14:6:30%:9.8E-1:31:25:5:9:2:4\n-chr1\t16534\t.\tC\tT\t.\tPASS\tADP=11;WT=0;HET=1;HOM=0;NC=0\tGT:GQ:SDP:DP:RD:AD:FREQ:PVAL:RBQ:ABQ:RDF:RDR:ADF:ADR\t0/1:0:11:11:6:4:40%:9.8E-1:32:37:1:5:0:4\n-chr1\t16571\t.\tG\tA\t.\tPASS\tADP=8;WT=0;HET=1;HOM=0;NC=0\tGT:GQ:SDP:DP:RD:AD:FREQ:PVAL:RBQ:ABQ:RDF:RDR:ADF:ADR\t0/1:0:8:8:5:2:28.57%:9.8E-1:31:34:2:3:0:2\n-chr1\t16737\t.\tG\tT\t.\tPASS\tADP=27;WT=0;HET=1;HOM=0;NC=0\tGT:GQ:SDP:DP:RD:AD:FREQ:PVAL:RBQ:ABQ:RDF:RDR:ADF:ADR\t0/1:0:30:27:22:5:18.52%:9.8E-1:37:34:21:1:5:0\n-chr1\t16742\t.\tG\tC\t.\tPASS\tADP=30;WT=0;HET=1;HOM=0;NC=0\tGT:GQ:SDP:DP:RD:AD:FREQ:PVAL:RBQ:ABQ:RDF:RDR:ADF:ADR\t0/1:0:30:30:23:7:23.33%:9.8E-1:32:37:21:2:7:0\n-chr1\t16900\t.\tT\tC\t.\tMinAF\tADP=140;WT=0;HET=1;HOM=0;NC=0\tGT:GQ:SDP:DP:RD:AD:FREQ:PVAL:RBQ:ABQ:RDF:RDR:ADF:ADR\t0/1:0:150:140:135:5:3.57%:9.8E-1:35:25:119:16:5:0\n-chr1\t16963\t.\tG\tA\t.\tPASS\tADP=338;WT=0;HET=1;HOM=0;NC=0\tGT:GQ:SDP:DP:RD:AD:FREQ:PVAL:RBQ:ABQ:RDF:RDR:ADF:ADR\t0/1:0:354:338:297:41:12.13%:9.8E-1:32:34:218:79:26:15\n-chr1\t16977\t.\tG\tA\t.\tPASS\tADP=368;WT=0;HET=1;HOM=0;NC=0\tGT:GQ:SDP:DP:RD:AD:FREQ:PVAL:RBQ:ABQ:RDF:RDR:ADF:ADR\t0/1:0:384:368:270:98:26.63%:9.8E-1:34:25:186:84:63:35\n-chr1\t16996\t.\tT\tC\t.\tPASS\tADP=356;WT=0;HET=1;HOM=0;NC=0\tGT:GQ:SDP:DP:RD:AD:FREQ:PVAL:RBQ:ABQ:RDF:RDR:ADF:ADR\t0/1:0:371:356:269:87:24.44%:9.8E-1:35:34:147:122:58:29\n-chr1\t17020\t.\tG\tA\t.\tMinAF\tADP=355;WT=0;HET=1;HOM=0;NC=0\tGT:GQ:SDP:DP:RD:AD:FREQ:PVAL:RBQ:ABQ:RDF:RDR:ADF:ADR\t0/1:0:380:355:347:8:2.25%:9.8E-1:33:29:164:183:4:4\n" ], "status": "failure", "test_index": 0, "time_seconds": 117.28618025779724, "tool_id": "toolshed.g2.bx.psu.edu/repos/devteam/vcftools_annotate/vcftools_annotate", "tool_version": "0.1"}
                ]]></error>
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            /galaxy/server/database/jobs_directory/000/22/tool_script.sh: line 22: vcf-annotate: command not found
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="toolshed.g2.bx.psu.edu/repos/devteam/vcftools_slice/vcftools_slice (Test #1)" name="0" time="149.18512177467346">
        
            
                <error type="error" message="Tool exit code: 0"><![CDATA[
                    { "inputs": {  "input": {   "id": "632036ec301168a1",   "src": "hda"  },  "regions": {   "id": "4cab9a558c771da7",   "src": "hda"  } }, "job": {  "command_line": "vcf-sort /galaxy/server/database/objects/7/b/0/dataset_7b03a0cc-9d5c-4dd9-a43b-250ce8e1828b.dat \u003e sorted.vcf  ; echo \u0027#dummy header\u0027 | cat - /galaxy/server/database/objects/e/c/1/dataset_ec1a1bed-cd67-42eb-bcc8-7b0375b96c5a.dat \u003e regions.bed  ; vcftools --vcf sorted.vcf --out output --bed regions.bed --recode",  "command_version": "",  "create_time": "2023-01-13T04:47:16.094886",  "dependencies": [],  "exit_code": 0,  "external_id": "gxy-5tzpv",  "galaxy_version": "21.09",  "history_id": "702fa106437bef85",  "id": "afed891f0227ab19",  "inputs": {   "input": {    "id": "632036ec301168a1",    "src": "hda",    "uuid": "7b03a0cc-9d5c-4dd9-a43b-250ce8e1828b"   },   "regions": {    "id": "4cab9a558c771da7",    "src": "hda",    "uuid": "ec1a1bed-cd67-42eb-bcc8-7b0375b96c5a"   }  },  "job_messages": [],  "job_metrics": [   {    "name": "galaxy_slots",    "plugin": "core",    "raw_value": "1.0000000",    "title": "Cores Allocated",    "value": "1"   },   {    "name": "galaxy_memory_mb",    "plugin": "core",    "raw_value": "2000.0000000",    "title": "Memory Allocated (MB)",    "value": "2000"   },   {    "name": "start_epoch",    "plugin": "core",    "raw_value": "1673585347.0000000",    "title": "Job Start Time",    "value": "2023-01-13 04:49:07"   },   {    "name": "end_epoch",    "plugin": "core",    "raw_value": "1673585347.0000000",    "title": "Job End Time",    "value": "2023-01-13 04:49:07"   },   {    "name": "runtime_seconds",    "plugin": "core",    "raw_value": "0E-7",    "title": "Job Runtime (Wall Clock)",    "value": "0 seconds"   }  ],  "job_stderr": "",  "job_stdout": "",  "model_class": "Job",  "output_collections": {},  "outputs": {   "output": {    "id": "afed891f0227ab19",    "src": "hda",    "uuid": "4994d752-ee11-4e47-9ef9-ecd594dcac4f"   }  },  "params": {   "__input_ext": "\"vcf\"",   "chromInfo": "\"/cvmfs/data.galaxyproject.org/managed/len/ucsc/?.len\"",   "dbkey": "\"?\""  },  "state": "error",  "stderr": "/galaxy/server/database/jobs_directory/000/33/tool_script.sh: line 22: vcf-sort: command not found\n",  "stdout": "\nVCFtools - v0.1.11\n(C) Adam Auton 2009\n\nParameters as interpreted:\n\t--vcf sorted.vcf\n\t--out output\n\t--recode\n\t--bed regions.bed\n\nIndex file is older than variant file. Will regenerate.\nBuilding new index file.\nError:No header or meta information. Invalid file: sorted.vcf\n",  "tool_id": "toolshed.g2.bx.psu.edu/repos/devteam/vcftools_slice/vcftools_slice/0.1",  "tool_stderr": "/galaxy/server/database/jobs_directory/000/33/tool_script.sh: line 22: vcf-sort: command not found\n",  "tool_stdout": "\nVCFtools - v0.1.11\n(C) Adam Auton 2009\n\nParameters as interpreted:\n\t--vcf sorted.vcf\n\t--out output\n\t--recode\n\t--bed regions.bed\n\nIndex file is older than variant file. Will regenerate.\nBuilding new index file.\nError:No header or meta information. Invalid file: sorted.vcf\n",  "traceback": null,  "update_time": "2023-01-13T04:49:18.495021",  "user_email": "tests@fake.org" }, "output_problems": [  "Job in error state.. tool_id: toolshed.g2.bx.psu.edu/repos/devteam/vcftools_slice/vcftools_slice/0.1, exit_code: 0, stderr: /galaxy/server/database/jobs_directory/000/33/tool_script.sh: line 22: vcf-sort: command not found\n.",  "Job in error state.. tool_id: toolshed.g2.bx.psu.edu/repos/devteam/vcftools_slice/vcftools_slice/0.1, exit_code: 0, stderr: /galaxy/server/database/jobs_directory/000/33/tool_script.sh: line 22: vcf-sort: command not found\n." ], "status": "failure", "test_index": 0, "time_seconds": 149.18512177467346, "tool_id": "toolshed.g2.bx.psu.edu/repos/devteam/vcftools_slice/vcftools_slice", "tool_version": "0.1"}
                ]]></error>
            
            <system-out><![CDATA[
            VCFtools - v0.1.11(C) Adam Auton 2009Parameters as interpreted:--vcf sorted.vcf--out output--recode--bed regions.bedIndex file is older than variant file. Will regenerate.Building new index file.Error:No header or meta information. Invalid file: sorted.vcf
            ]]></system-out>
            <system-err><![CDATA[
            /galaxy/server/database/jobs_directory/000/33/tool_script.sh: line 22: vcf-sort: command not found
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="toolshed.g2.bx.psu.edu/repos/devteam/vcfvcfintersect/vcfvcfintersect (Test #1)" name="0" time="161.84720540046692">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            index file localref.fa.fai not found, generating...
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="toolshed.g2.bx.psu.edu/repos/devteam/vcfvcfintersect/vcfvcfintersect (Test #2)" name="1" time="159.60063648223877">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            index file localref.fa.fai not found, generating...
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="toolshed.g2.bx.psu.edu/repos/iuc/medaka_consensus/medaka_consensus (Test #1)" name="0" time="132.46529412269592">
        
            
            <system-out><![CDATA[
            [04:56:10 - Predict] Processing region(s): ref:0-30[04:56:10 - DataStore] Writing sample registry.[04:56:10 - Predict] Setting tensorflow threads to 1.2023-01-13 04:56:10.219620: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA2023-01-13 04:56:10.226512: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2200165000 Hz2023-01-13 04:56:10.227819: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x5590cc8bc440 executing computations on platform Host. Devices:2023-01-13 04:56:10.227909: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): <undefined>, <undefined>[04:56:10 - Predict] Processing 1 long region(s) with batching.[04:56:10 - Predict] Using model: /usr/local/lib/python3.6/site-packages/medaka/data/r941_prom_high_g360_model.hdf5.[04:56:10 - ModelLoad] Building model with cudnn optimization: False2023-01-13 04:56:11.378111: W tensorflow/compiler/jit/mark_for_compilation_pass.cc:1412] (One-time warning): Not using XLA:CPU for cluster because envvar TF_XLA_FLAGS=--tf_xla_cpu_global_jit was not set.  If you want XLA:CPU, either set that envvar, or use experimental_jit_scope to enable XLA:CPU.  To confirm that XLA is active, pass --vmodule=xla_compilation_cache=1 (as a proper command-line flag, not via TF_XLA_FLAGS) or set the envvar XLA_FLAGS=--xla_hlo_profile.[04:56:11 - DLoader] Initializing data loader[04:56:11 - DLoader] Submitting ref:0-30.[04:56:11 - PWorker] Running inference for 0.0M draft bases.[04:56:11 - Sampler] Initializing sampler for consensus of region ref:0-30.[04:56:11 - Feature] Pileup counts do not span requested region, requested ref:0-30, received 0-9.[04:56:11 - Feature] Processed ref:0.0-9.0 (median depth 1.0)[04:56:11 - Feature] Pileup counts do not span requested region, requested ref:0-30, received 14-29.[04:56:11 - Feature] Processed ref:14.0-29.0 (median depth 1.0)[04:56:11 - Sampler] Took 0.00s to make features.[04:56:11 - Sampler] Region ref:0.0-9.0 (10 positions) is smaller than inference chunk length 10000, quarantining.[04:56:11 - Sampler] Region ref:14.0-29.0 (16 positions) is smaller than inference chunk length 10000, quarantining.[04:56:11 - DataStore] Writing sample registry.[04:56:11 - PWorker] All done, 2 remainder regions.[04:56:11 - Predict] Processing 2 short region(s).[04:56:11 - ModelLoad] Building model with cudnn optimization: False[04:56:12 - DLoader] Initializing data loader[04:56:12 - DLoader] Submitting ref:0-10.[04:56:12 - PWorker] Running inference for 0.0M draft bases.[04:56:12 - Sampler] Initializing sampler for consensus of region ref:0-10.[04:56:12 - Feature] Processed ref:0.0-9.0 (median depth 1.0)[04:56:12 - Sampler] Took 0.00s to make features.[04:56:12 - Sampler] Pileup for ref:0.0-9.0 is of width 10[04:56:13 - DataStore] Writing sample registry.[04:56:13 - PWorker] All done, 0 remainder regions.[04:56:13 - DLoader] Initializing data loader[04:56:13 - DLoader] Submitting ref:14-30.[04:56:13 - PWorker] Running inference for 0.0M draft bases.[04:56:13 - Sampler] Initializing sampler for consensus of region ref:14-30.[04:56:13 - Feature] Processed ref:14.0-29.0 (median depth 1.0)[04:56:13 - Sampler] Took 0.00s to make features.[04:56:13 - Sampler] Pileup for ref:14.0-29.0 is of width 16[04:56:13 - DataStore] Writing sample registry.[04:56:13 - PWorker] All done, 0 remainder regions.[04:56:13 - Predict] Finished processing all regions.
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="toolshed.g2.bx.psu.edu/repos/iuc/medaka_consensus/medaka_consensus (Test #2)" name="1" time="119.17218899726868">
        
            
            <system-out><![CDATA[
            [04:56:14 - Predict] Processing region(s): ref:0-30[04:56:14 - DataStore] Writing sample registry.[04:56:14 - Predict] Setting tensorflow threads to 1.2023-01-13 04:56:14.835967: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA2023-01-13 04:56:14.840445: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2200165000 Hz2023-01-13 04:56:14.841606: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x55c701ae5f40 executing computations on platform Host. Devices:2023-01-13 04:56:14.841642: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): <undefined>, <undefined>[04:56:14 - Predict] Processing 1 long region(s) with batching.[04:56:14 - Predict] Using model: /usr/local/lib/python3.6/site-packages/medaka/data/r941_min_fast_g303_model.hdf5.[04:56:14 - ModelLoad] Building model with cudnn optimization: False2023-01-13 04:56:15.936070: W tensorflow/compiler/jit/mark_for_compilation_pass.cc:1412] (One-time warning): Not using XLA:CPU for cluster because envvar TF_XLA_FLAGS=--tf_xla_cpu_global_jit was not set.  If you want XLA:CPU, either set that envvar, or use experimental_jit_scope to enable XLA:CPU.  To confirm that XLA is active, pass --vmodule=xla_compilation_cache=1 (as a proper command-line flag, not via TF_XLA_FLAGS) or set the envvar XLA_FLAGS=--xla_hlo_profile.[04:56:16 - DLoader] Initializing data loader[04:56:16 - DLoader] Submitting ref:0-30.[04:56:16 - PWorker] Running inference for 0.0M draft bases.[04:56:16 - Sampler] Initializing sampler for consensus of region ref:0-30.[04:56:16 - Feature] Pileup counts do not span requested region, requested ref:0-30, received 0-9.[04:56:16 - Feature] Processed ref:0.0-9.0 (median depth 1.0)[04:56:16 - Feature] Pileup counts do not span requested region, requested ref:0-30, received 14-29.[04:56:16 - Feature] Processed ref:14.0-29.0 (median depth 1.0)[04:56:16 - Sampler] Took 0.00s to make features.[04:56:16 - Sampler] Region ref:0.0-9.0 (10 positions) is smaller than inference chunk length 9999, quarantining.[04:56:16 - Sampler] Region ref:14.0-29.0 (16 positions) is smaller than inference chunk length 9999, quarantining.[04:56:16 - DataStore] Writing sample registry.[04:56:16 - PWorker] All done, 2 remainder regions.[04:56:16 - Predict] Processing 2 short region(s).[04:56:16 - ModelLoad] Building model with cudnn optimization: False[04:56:17 - DLoader] Initializing data loader[04:56:17 - DLoader] Submitting ref:0-10.[04:56:17 - PWorker] Running inference for 0.0M draft bases.[04:56:17 - Sampler] Initializing sampler for consensus of region ref:0-10.[04:56:17 - Feature] Processed ref:0.0-9.0 (median depth 1.0)[04:56:17 - Sampler] Took 0.00s to make features.[04:56:17 - Sampler] Pileup for ref:0.0-9.0 is of width 10[04:56:17 - DataStore] Writing sample registry.[04:56:17 - PWorker] All done, 0 remainder regions.[04:56:17 - DLoader] Initializing data loader[04:56:17 - DLoader] Submitting ref:14-30.[04:56:17 - PWorker] Running inference for 0.0M draft bases.[04:56:17 - Sampler] Initializing sampler for consensus of region ref:14-30.[04:56:17 - Feature] Processed ref:14.0-29.0 (median depth 1.0)[04:56:17 - Sampler] Took 0.00s to make features.[04:56:17 - Sampler] Pileup for ref:14.0-29.0 is of width 16[04:56:17 - DataStore] Writing sample registry.[04:56:17 - PWorker] All done, 0 remainder regions.[04:56:17 - Predict] Finished processing all regions.[04:56:17 - Predict] Validating and finalising output data.[04:56:17 - DataStore] Writing sample registry.
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="toolshed.g2.bx.psu.edu/repos/iuc/medaka_consensus_pipeline/medaka_consensus_pipeline (Test #1)" name="0" time="162.47041535377502">
        
            
            <system-out><![CDATA[
            Checking program versionsThis is medaka 1.0.3Program    Version    Required   Pass     bcftools   1.10.2     1.9        True     bgzip      1.10.2     1.9        True     minimap2   2.17       2.11       True     samtools   1.10       1.9        True     tabix      1.10.2     1.9        True     Aligning basecalls to draftRemoving previous index file /galaxy/server/database/objects/9/8/c/dataset_98c5072b-200f-4ed3-9bee-4a0214ee7732.dat.mmiRemoving previous index file /galaxy/server/database/objects/9/8/c/dataset_98c5072b-200f-4ed3-9bee-4a0214ee7732.dat.faiConstructing minimap index.[M::mm_idx_gen::0.003*0.97] collected minimizers[M::mm_idx_gen::0.004*1.15] sorted minimizers[M::main::0.006*1.11] loaded/built the index for 1 target sequence(s)[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 1[M::mm_idx_stat::0.006*1.10] distinct minimizers: 688 (100.00% are singletons); average occurrences: 1.000; average spacing: 5.512[M::main] Version: 2.17-r941[M::main] CMD: minimap2 -I 16G -x map-ont --MD -d /galaxy/server/database/objects/9/8/c/dataset_98c5072b-200f-4ed3-9bee-4a0214ee7732.dat.mmi /galaxy/server/database/objects/9/8/c/dataset_98c5072b-200f-4ed3-9bee-4a0214ee7732.dat[M::main] Real time: 0.028 sec; CPU: 0.007 sec; Peak RSS: 0.003 GB[M::main::0.003*1.12] loaded/built the index for 1 target sequence(s)[M::mm_mapopt_update::0.003*1.11] mid_occ = 2[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 1[M::mm_idx_stat::0.004*1.10] distinct minimizers: 688 (100.00% are singletons); average occurrences: 1.000; average spacing: 5.512[M::worker_pipeline::0.148*0.92] mapped 125 sequences[M::main] Version: 2.17-r941[M::main] CMD: minimap2 -x map-ont --MD -t 1 -a -A 2 -B 4 -O 4,24 -E 2,1 /galaxy/server/database/objects/9/8/c/dataset_98c5072b-200f-4ed3-9bee-4a0214ee7732.dat.mmi /galaxy/server/database/objects/a/a/1/dataset_aa1cbecf-4529-470c-837d-018486b90aea.dat[M::main] Real time: 0.150 sec; CPU: 0.138 sec; Peak RSS: 0.008 GBRunning medaka consensus[04:55:31 - Predict] Processing region(s): 4:0-3792[04:55:31 - Predict] Setting tensorflow threads to 1.[04:55:31 - Predict] Processing 1 long region(s) with batching.[04:55:31 - Predict] Using model: /usr/local/lib/python3.6/site-packages/medaka/data/r941_prom_high_g360_model.hdf5.[04:55:31 - ModelLoad] Building model with cudnn optimization: False[04:55:32 - DLoader] Initializing data loader[04:55:32 - PWorker] Running inference for 0.0M draft bases.[04:55:32 - Sampler] Initializing sampler for consensus of region 4:0-3792.[04:55:32 - Feature] Processed 4:0.0-3791.0 (median depth 78.0)[04:55:32 - Sampler] Took 0.02s to make features.[04:55:32 - Sampler] Region 4:0.0-3791.0 (7485 positions) is smaller than inference chunk length 10000, quarantining.[04:55:32 - PWorker] All done, 1 remainder regions.[04:55:32 - Predict] Processing 1 short region(s).[04:55:32 - ModelLoad] Building model with cudnn optimization: False[04:55:33 - DLoader] Initializing data loader[04:55:33 - PWorker] Running inference for 0.0M draft bases.[04:55:33 - Sampler] Initializing sampler for consensus of region 4:0-3792.[04:55:33 - Feature] Processed 4:0.0-3791.0 (median depth 78.0)[04:55:33 - Sampler] Took 0.02s to make features.[04:55:38 - PWorker] All done, 0 remainder regions.[04:55:38 - Predict] Finished processing all regions.Using medaka stitch to create consensus.[04:55:40 - DataIndex] Loaded 1/1 (100.00%) sample files.[04:55:40 - Stitch] Stitching regions: ['4:0-'][04:55:40 - DataIndex] Loaded 1/1 (100.00%) sample files.[04:55:40 - Stitch] Processing 4:0-.[04:55:40 - Stitch] Used heuristic 0 times for 4:0-.Polished assembly written to results/consensus.fasta, have a nice day.
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="toolshed.g2.bx.psu.edu/repos/iuc/medaka_consensus_pipeline/medaka_consensus_pipeline (Test #2)" name="1" time="152.4656684398651">
        
            
            <system-out><![CDATA[
            Checking program versionsThis is medaka 1.0.3Program    Version    Required   Pass     bcftools   1.10.2     1.9        True     bgzip      1.10.2     1.9        True     minimap2   2.17       2.11       True     samtools   1.10       1.9        True     tabix      1.10.2     1.9        True     Aligning basecalls to draftRemoving previous index file /galaxy/server/database/objects/0/7/4/dataset_074b3d9d-6fdb-4818-970b-2146967ed0dc.dat.mmiRemoving previous index file /galaxy/server/database/objects/0/7/4/dataset_074b3d9d-6fdb-4818-970b-2146967ed0dc.dat.faiConstructing minimap index.[M::mm_idx_gen::0.004*0.90] collected minimizers[M::mm_idx_gen::0.005*1.17] sorted minimizers[M::main::0.007*1.13] loaded/built the index for 1 target sequence(s)[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 1[M::mm_idx_stat::0.007*1.13] distinct minimizers: 688 (100.00% are singletons); average occurrences: 1.000; average spacing: 5.512[M::main] Version: 2.17-r941[M::main] CMD: minimap2 -I 16G -x map-ont --MD -d /galaxy/server/database/objects/0/7/4/dataset_074b3d9d-6fdb-4818-970b-2146967ed0dc.dat.mmi /galaxy/server/database/objects/0/7/4/dataset_074b3d9d-6fdb-4818-970b-2146967ed0dc.dat[M::main] Real time: 0.020 sec; CPU: 0.008 sec; Peak RSS: 0.003 GB[M::main::0.004*0.91] loaded/built the index for 1 target sequence(s)[M::mm_mapopt_update::0.005*0.90] mid_occ = 2[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 1[M::mm_idx_stat::0.005*0.90] distinct minimizers: 688 (100.00% are singletons); average occurrences: 1.000; average spacing: 5.512[M::worker_pipeline::0.159*0.88] mapped 125 sequences[M::main] Version: 2.17-r941[M::main] CMD: minimap2 -x map-ont --MD -t 1 -a -A 2 -B 4 -O 4,24 -E 2,1 /galaxy/server/database/objects/0/7/4/dataset_074b3d9d-6fdb-4818-970b-2146967ed0dc.dat.mmi /galaxy/server/database/objects/3/2/b/dataset_32b9c6fe-ede3-4523-a682-cecf18b91152.dat[M::main] Real time: 0.161 sec; CPU: 0.141 sec; Peak RSS: 0.008 GBRunning medaka consensus[04:56:09 - Predict] Processing region(s): 4:0-3792[04:56:09 - Predict] Setting tensorflow threads to 1.[04:56:09 - Predict] Processing 1 long region(s) with batching.[04:56:09 - Predict] Using model: /usr/local/lib/python3.6/site-packages/medaka/data/r941_min_fast_g303_model.hdf5.[04:56:09 - ModelLoad] Building model with cudnn optimization: False[04:56:11 - DLoader] Initializing data loader[04:56:11 - PWorker] Running inference for 0.0M draft bases.[04:56:11 - Sampler] Initializing sampler for consensus of region 4:0-3792.[04:56:11 - Feature] Processed 4:0.0-3791.0 (median depth 78.0)[04:56:11 - Sampler] Took 0.02s to make features.[04:56:11 - Sampler] Region 4:0.0-3791.0 (7485 positions) is smaller than inference chunk length 10000, quarantining.[04:56:11 - PWorker] All done, 1 remainder regions.[04:56:11 - Predict] Processing 1 short region(s).[04:56:11 - ModelLoad] Building model with cudnn optimization: False[04:56:12 - DLoader] Initializing data loader[04:56:12 - Sampler] Initializing sampler for consensus of region 4:0-3792.[04:56:12 - PWorker] Running inference for 0.0M draft bases.[04:56:12 - Feature] Processed 4:0.0-3791.0 (median depth 78.0)[04:56:12 - Sampler] Took 0.02s to make features.[04:56:17 - PWorker] All done, 0 remainder regions.[04:56:17 - Predict] Finished processing all regions.Using medaka variant to create vcf and apply to draft.[04:56:19 - DataIndex] Loaded 1/1 (100.00%) sample files.[04:56:19 - Variants] Processing 4:0-.Applying variantsApplied 4 variantsVCF with changes made by medaka: results/variants.vcf.gz[04:56:22 - DataIndex] Loaded 1/1 (100.00%) sample files.[04:56:22 - Variants] Building interval tree[04:56:22 - Variants] Writing intervals for 4[04:56:22 - Variants] All done, bed file written to polished_regions_in_draft_coords.bedBED delineating regions which were polished (in draft coordinates): results/polished_regions_in_draft_coords.bedPolished assembly written to results/consensus.fasta, have a nice day.
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="toolshed.g2.bx.psu.edu/repos/iuc/medaka_variant/medaka_variant (Test #1)" name="0" time="193.3935260772705">
        
            
            <system-out><![CDATA[
            [04:49:44 - DataIndex] Loaded 1/2 (50.00%) sample files.[04:49:44 - DataIndex] Loaded 2/2 (100.00%) sample files.[04:49:44 - Variants] Label decoding is:0: ('*',)1: ('A',)2: ('C',)3: ('G',)4: ('T',)[04:49:44 - Variants] Processing NC_045512.2:0-.[04:49:44 - TrimOverlap] NC_045512.2:30.0-2939.0 is contained within NC_045512.2:30.0-2939.0, skipping.
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="toolshed.g2.bx.psu.edu/repos/iuc/medaka_variant/medaka_variant (Test #2)" name="1" time="173.51845717430115">
        
            
            <system-out><![CDATA[
            [04:49:43 - DataIndex] Loaded 1/1 (100.00%) sample files.[04:49:43 - Variants] Label decoding is:0: ('*',)1: ('A',)2: ('C',)3: ('G',)4: ('T',)[04:49:43 - Variants] Processing NC_045512.2:0-.
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="toolshed.g2.bx.psu.edu/repos/iuc/medaka_variant/medaka_variant (Test #3)" name="2" time="172.53824496269226">
        
            
            <system-out><![CDATA[
            [04:49:44 - DataIndex] Loaded 1/1 (100.00%) sample files.[04:49:44 - Variants] Label decoding is:0: ('*',)1: ('A',)2: ('C',)3: ('G',)4: ('T',)[04:49:44 - Variants] Processing NC_045512.2:0-.
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="toolshed.g2.bx.psu.edu/repos/iuc/medaka_variant_pipeline/medaka_variant_pipeline (Test #1)" name="0" time="23.362857341766357">
        
            <error type="error" message="parameter 'ref_file': an invalid option ('bwa-mem-mt-genome') was selected (valid options: calJac3,Spur_v2.6,danRer3,Hydra_JCVI,ce2,hetGla1,rheMac3,Zea_mays_B73_RefGen_v2,eriEur1,burkPseu_668,monDom5,galGal3,felCat5,bosTauMd3,eschColi_O157H7,petMar1,caePb2,ce4,rheMac2,canFam1,xenTro2,Homo_sapiens_nuHg19_mtrCRS,chrPic1,otoGar1,danRer10,hg17canon,dipOrd1,rn3canon,droSec1,hg18chr21,eschColi_K12,vicPac2,rn6,Xanthomonas_oryzae_KACC10331,dm6,mm7canon,CHM13_T2T_v2.0,geoFor1,hg19_rCRS_pUC18_phiX174,gasAcu1,sacCer1,papHam1,aplCal1,galGal2,oryLat2,latCha1,hg_g1k_v37,felCat4canon,Tcas_3.0,aptMan1,bosTau8,Araly1,ce8,mm9canon,oryCun2,caeJap1,dm3,sarHar1,sacCer2,baciSubt,apiMel4,gorGor3,Arabidopsis_thaliana_TAIR9,papAnu2,eschColi_O157H7_EDL933,panTro2,hg18,anoCar2,tetNig1,rn4canon,AaegL1,Amel_4.5,hg38canon,ce3,Schizosaccharomyces_pombe_1.1,apiMel3,fr1,strePneu_R6,hg17,felCat3,dasNov3,loxAfr1,rn5,Arabidopsis_thaliana_TAIR10,ce7,oryCun1,droPer1,hg18nohaps,galGal2canon,hg19canon,strPur2,danRer4,canFam2,caeRem1,galGal4,droSim1,nomLeu3,ce9,lMaj5,musFur1,monDom1,droAna2,myoLuc2,susScr1,Xanthomonas_oryzae_PXO99A,vicPac1,cb3canon,venter1,dp3,loxAfr3,felCat8,droMoj2,eschColi_APEC_O1,dp4,burkPseu_K96243,rn3,priPac1,susScr2,mm8,taeGut1,mm9female,panTro3full,Tcacao_1.0,cavPor3,anoCar1,Ptrichocarpa_156,felCat4,borEut13,criGri1,ci2canon,droYak2,pUC18,bosTau3,cerSim1,burkPseu_1106A,eschColi_CFT073,hg19chrM,CpipJ1,taeGut2,droMoj3,danRer6,droWil1,droMoj1,droGri1,micMur1,panTro2canon,saiBol1,galGal3canon,melGal1,Bombyx_mori_p50T_2.0,calJac3canon,mm8canon,droYak1,danRer7,anoGam1,nomLeu2,oryza_sativa_japonica_nipponbare_IRGSP4.0,cb3full,caePb3,sorAra1,equCab2chrM,speTri2,danRer5,pseuAeru_PAO1,triCas2,droEre1,bosTau2,pseuAeru_PA7,rn4,turTru2,caeJap4,hg19female,ce5,xenTro1,otoGar3,eschColi_K12_DH10B,hg38,PhumU1,ente638,droAna1,fr2,Sscrofa9.58,strPur3,panPan1,caePb1,caeRem2,Xanthomonas_oryzae_MAFF_311018,bordBron,bosTau5,IscaW1,apiMel2,ornAna1,tetNig2,caeRem3,tarSyr1,ci2full,eschColi_O157H7_EC4115,burkPseu_1710B,panTro1canon,dm2,ponAbe2,allMis1,hg18canon,ailMel1,caeJap2,panTro1,canFam3,droAna3,apiMel1,melUnd1,panTro3canon,Homo_sapiens_AK1,oryLat2canon,braFlo1,monDom4,caeRem4,equCab1,hg38female,27835,mm9,equCab2,eschColi_O157H7_TW14359,susScr3,phiX,hg19,danRer2,mm10,bosTau7,droVir1,xenTro3,droVir2,sacCer3,calJac1,panTro4,hg19chrX,oviAri1,bosTau6,droGri2,oryLat1,nomLeu1,tetNig2canon,dasNov1,gadMor1,eschColi_K12_W3110,ce6,caeJap3,bosTau4,ce10,tetNig1canon,AgamP3,pseuAeru_UCBPP_PA14,echTel1,droVir3,droEre2,tupBel1,mm7,choHof1,taeGut1canon)"><![CDATA[
                { "dynamic_param_error": true, "execution_problem": "parameter \u0027ref_file\u0027: an invalid option (\u0027bwa-mem-mt-genome\u0027) was selected (valid options: calJac3,Spur_v2.6,danRer3,Hydra_JCVI,ce2,hetGla1,rheMac3,Zea_mays_B73_RefGen_v2,eriEur1,burkPseu_668,monDom5,galGal3,felCat5,bosTauMd3,eschColi_O157H7,petMar1,caePb2,ce4,rheMac2,canFam1,xenTro2,Homo_sapiens_nuHg19_mtrCRS,chrPic1,otoGar1,danRer10,hg17canon,dipOrd1,rn3canon,droSec1,hg18chr21,eschColi_K12,vicPac2,rn6,Xanthomonas_oryzae_KACC10331,dm6,mm7canon,CHM13_T2T_v2.0,geoFor1,hg19_rCRS_pUC18_phiX174,gasAcu1,sacCer1,papHam1,aplCal1,galGal2,oryLat2,latCha1,hg_g1k_v37,felCat4canon,Tcas_3.0,aptMan1,bosTau8,Araly1,ce8,mm9canon,oryCun2,caeJap1,dm3,sarHar1,sacCer2,baciSubt,apiMel4,gorGor3,Arabidopsis_thaliana_TAIR9,papAnu2,eschColi_O157H7_EDL933,panTro2,hg18,anoCar2,tetNig1,rn4canon,AaegL1,Amel_4.5,hg38canon,ce3,Schizosaccharomyces_pombe_1.1,apiMel3,fr1,strePneu_R6,hg17,felCat3,dasNov3,loxAfr1,rn5,Arabidopsis_thaliana_TAIR10,ce7,oryCun1,droPer1,hg18nohaps,galGal2canon,hg19canon,strPur2,danRer4,canFam2,caeRem1,galGal4,droSim1,nomLeu3,ce9,lMaj5,musFur1,monDom1,droAna2,myoLuc2,susScr1,Xanthomonas_oryzae_PXO99A,vicPac1,cb3canon,venter1,dp3,loxAfr3,felCat8,droMoj2,eschColi_APEC_O1,dp4,burkPseu_K96243,rn3,priPac1,susScr2,mm8,taeGut1,mm9female,panTro3full,Tcacao_1.0,cavPor3,anoCar1,Ptrichocarpa_156,felCat4,borEut13,criGri1,ci2canon,droYak2,pUC18,bosTau3,cerSim1,burkPseu_1106A,eschColi_CFT073,hg19chrM,CpipJ1,taeGut2,droMoj3,danRer6,droWil1,droMoj1,droGri1,micMur1,panTro2canon,saiBol1,galGal3canon,melGal1,Bombyx_mori_p50T_2.0,calJac3canon,mm8canon,droYak1,danRer7,anoGam1,nomLeu2,oryza_sativa_japonica_nipponbare_IRGSP4.0,cb3full,caePb3,sorAra1,equCab2chrM,speTri2,danRer5,pseuAeru_PAO1,triCas2,droEre1,bosTau2,pseuAeru_PA7,rn4,turTru2,caeJap4,hg19female,ce5,xenTro1,otoGar3,eschColi_K12_DH10B,hg38,PhumU1,ente638,droAna1,fr2,Sscrofa9.58,strPur3,panPan1,caePb1,caeRem2,Xanthomonas_oryzae_MAFF_311018,bordBron,bosTau5,IscaW1,apiMel2,ornAna1,tetNig2,caeRem3,tarSyr1,ci2full,eschColi_O157H7_EC4115,burkPseu_1710B,panTro1canon,dm2,ponAbe2,allMis1,hg18canon,ailMel1,caeJap2,panTro1,canFam3,droAna3,apiMel1,melUnd1,panTro3canon,Homo_sapiens_AK1,oryLat2canon,braFlo1,monDom4,caeRem4,equCab1,hg38female,27835,mm9,equCab2,eschColi_O157H7_TW14359,susScr3,phiX,hg19,danRer2,mm10,bosTau7,droVir1,xenTro3,droVir2,sacCer3,calJac1,panTro4,hg19chrX,oviAri1,bosTau6,droGri2,oryLat1,nomLeu1,tetNig2canon,dasNov1,gadMor1,eschColi_K12_W3110,ce6,caeJap3,bosTau4,ce10,tetNig1canon,AgamP3,pseuAeru_UCBPP_PA14,echTel1,droVir3,droEre2,tupBel1,mm7,choHof1,taeGut1canon)", "inputs": {  "i": {   "id": "c90a16698adb7dc8",   "src": "hda"  },  "out": [   "round_0_hap_mixed_probs_hdf",   "round_0_hap_mixed_unphased_vcf",   "log"  ],  "reference_source|ref_file": "bwa-mem-mt-genome",  "reference_source|reference_source_selector": "cached" }, "status": "error", "test_index": 0, "time_seconds": 23.362857341766357, "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/medaka_variant_pipeline/medaka_variant_pipeline", "tool_version": "1.0.3+galaxy2"}
            ]]></error>
        
    </testcase>
    
    <testcase classname="toolshed.g2.bx.psu.edu/repos/iuc/nanoplot/nanoplot (Test #1)" name="0" time="166.37294387817383">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="toolshed.g2.bx.psu.edu/repos/iuc/nanoplot/nanoplot (Test #2)" name="1" time="164.2270221710205">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="toolshed.g2.bx.psu.edu/repos/iuc/nanoplot/nanoplot (Test #3)" name="2" time="131.96647357940674">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="toolshed.g2.bx.psu.edu/repos/iuc/poretools_events/poretools_events (Test #1)" name="0" time="149.46808314323425">
        
            
                <error type="error" message="Tool exit code: 1"><![CDATA[
                    { "inputs": {  "input": {   "id": "5abe9213e1aaabcd",   "src": "hda"  },  "precalled": false }, "job": {  "command_line": "poretools events  \u0027/galaxy/server/database/objects/7/7/9/dataset_779c8fd4-76af-4aea-aef9-a2317f3f4512.dat\u0027 \u003e \u0027/galaxy/server/database/objects/8/4/f/dataset_84f9ce41-9428-42a8-8ca7-6c1747c300e1.dat\u0027",  "command_version": "",  "create_time": "2023-01-13T04:56:57.788556",  "dependencies": [],  "exit_code": 1,  "external_id": "gxy-z5lm2",  "galaxy_version": "21.09",  "history_id": "702fa106437bef85",  "id": "1307b51ef5524e63",  "inputs": {   "input": {    "id": "5abe9213e1aaabcd",    "src": "hda",    "uuid": "779c8fd4-76af-4aea-aef9-a2317f3f4512"   }  },  "job_messages": [   {    "code_desc": "",    "desc": "Fatal error: Exit code 1 ()",    "error_level": 3,    "exit_code": 1,    "type": "exit_code"   }  ],  "job_metrics": [   {    "name": "runtime_seconds",    "plugin": "core",    "raw_value": "1.0000000",    "title": "Job Runtime (Wall Clock)",    "value": "1 second"   },   {    "name": "end_epoch",    "plugin": "core",    "raw_value": "1673585929.0000000",    "title": "Job End Time",    "value": "2023-01-13 04:58:49"   },   {    "name": "start_epoch",    "plugin": "core",    "raw_value": "1673585928.0000000",    "title": "Job Start Time",    "value": "2023-01-13 04:58:48"   },   {    "name": "galaxy_memory_mb",    "plugin": "core",    "raw_value": "2000.0000000",    "title": "Memory Allocated (MB)",    "value": "2000"   },   {    "name": "galaxy_slots",    "plugin": "core",    "raw_value": "1.0000000",    "title": "Cores Allocated",    "value": "1"   }  ],  "job_stderr": "Traceback (most recent call last):\n  File \"/usr/local/bin/poretools\", line 11, in \u003cmodule\u003e\n    load_entry_point(\u0027poretools==0.6.0\u0027, \u0027console_scripts\u0027, \u0027poretools\u0027)()\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 543, in main\n    args.func(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 55, in run_subtool\n    submodule.run(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/events.py\", line 17, in run\n    for fast5 in Fast5File.Fast5FileSet(args.files):\n  File \"/usr/local/lib/python2.7/site-packages/poretools/Fast5File.py\", line 110, in next\n    shutil.rmtree(PORETOOLS_TMPDIR)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 261, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 270, in rmtree\n    onerror(os.rmdir, path, sys.exc_info())\n  File \"/usr/local/lib/python2.7/shutil.py\", line 268, in rmtree\n    os.rmdir(path)\nOSError: [Errno 39] Directory not empty: \u0027.poretools_tmp/test-data\u0027\n",  "job_stdout": "",  "model_class": "Job",  "output_collections": {},  "outputs": {   "output": {    "id": "91c14fe437f1e2ac",    "src": "hda",    "uuid": "84f9ce41-9428-42a8-8ca7-6c1747c300e1"   }  },  "params": {   "__input_ext": "\"fast5.tar\"",   "chromInfo": "\"/cvmfs/data.galaxyproject.org/managed/len/ucsc/?.len\"",   "dbkey": "\"?\"",   "precalled": "\"false\""  },  "state": "error",  "stderr": "\nTraceback (most recent call last):\n  File \"/usr/local/bin/poretools\", line 11, in \u003cmodule\u003e\n    load_entry_point(\u0027poretools==0.6.0\u0027, \u0027console_scripts\u0027, \u0027poretools\u0027)()\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 543, in main\n    args.func(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 55, in run_subtool\n    submodule.run(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/events.py\", line 17, in run\n    for fast5 in Fast5File.Fast5FileSet(args.files):\n  File \"/usr/local/lib/python2.7/site-packages/poretools/Fast5File.py\", line 110, in next\n    shutil.rmtree(PORETOOLS_TMPDIR)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 261, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 270, in rmtree\n    onerror(os.rmdir, path, sys.exc_info())\n  File \"/usr/local/lib/python2.7/shutil.py\", line 268, in rmtree\n    os.rmdir(path)\nOSError: [Errno 39] Directory not empty: \u0027.poretools_tmp/test-data\u0027\n",  "stdout": "",  "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/poretools_events/poretools_events/0.6.1a1.1",  "tool_stderr": "",  "tool_stdout": "",  "traceback": null,  "update_time": "2023-01-13T04:59:08.290241",  "user_email": "tests@fake.org" }, "output_problems": [  "Job in error state.. tool_id: toolshed.g2.bx.psu.edu/repos/iuc/poretools_events/poretools_events/0.6.1a1.1, exit_code: 1, stderr: \nTraceback (most recent call last):\n  File \"/usr/local/bin/poretools\", line 11, in \u003cmodule\u003e\n    load_entry_point(\u0027poretools==0.6.0\u0027, \u0027console_scripts\u0027, \u0027poretools\u0027)()\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 543, in main\n    args.func(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 55, in run_subtool\n    submodule.run(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/events.py\", line 17, in run\n    for fast5 in Fast5File.Fast5FileSet(args.files):\n  File \"/usr/local/lib/python2.7/site-packages/poretools/Fast5File.py\", line 110, in next\n    shutil.rmtree(PORETOOLS_TMPDIR)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 261, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 270, in rmtree\n    onerror(os.rmdir, path, sys.exc_info())\n  File \"/usr/local/lib/python2.7/shutil.py\", line 268, in rmtree\n    os.rmdir(path)\nOSError: [Errno 39] Directory not empty: \u0027.poretools_tmp/test-data\u0027\n.",  "Job in error state.. tool_id: toolshed.g2.bx.psu.edu/repos/iuc/poretools_events/poretools_events/0.6.1a1.1, exit_code: 1, stderr: \nTraceback (most recent call last):\n  File \"/usr/local/bin/poretools\", line 11, in \u003cmodule\u003e\n    load_entry_point(\u0027poretools==0.6.0\u0027, \u0027console_scripts\u0027, \u0027poretools\u0027)()\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 543, in main\n    args.func(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 55, in run_subtool\n    submodule.run(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/events.py\", line 17, in run\n    for fast5 in Fast5File.Fast5FileSet(args.files):\n  File \"/usr/local/lib/python2.7/site-packages/poretools/Fast5File.py\", line 110, in next\n    shutil.rmtree(PORETOOLS_TMPDIR)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 261, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 270, in rmtree\n    onerror(os.rmdir, path, sys.exc_info())\n  File \"/usr/local/lib/python2.7/shutil.py\", line 268, in rmtree\n    os.rmdir(path)\nOSError: [Errno 39] Directory not empty: \u0027.poretools_tmp/test-data\u0027\n." ], "status": "failure", "test_index": 0, "time_seconds": 149.46808314323425, "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/poretools_events/poretools_events", "tool_version": "0.6.1a1.1"}
                ]]></error>
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            Traceback (most recent call last):  File "/usr/local/bin/poretools", line 11, in <module>    load_entry_point('poretools==0.6.0', 'console_scripts', 'poretools')()  File "/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py", line 543, in main    args.func(parser, args)  File "/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py", line 55, in run_subtool    submodule.run(parser, args)  File "/usr/local/lib/python2.7/site-packages/poretools/events.py", line 17, in run    for fast5 in Fast5File.Fast5FileSet(args.files):  File "/usr/local/lib/python2.7/site-packages/poretools/Fast5File.py", line 110, in next    shutil.rmtree(PORETOOLS_TMPDIR)  File "/usr/local/lib/python2.7/shutil.py", line 261, in rmtree    rmtree(fullname, ignore_errors, onerror)  File "/usr/local/lib/python2.7/shutil.py", line 270, in rmtree    onerror(os.rmdir, path, sys.exc_info())  File "/usr/local/lib/python2.7/shutil.py", line 268, in rmtree    os.rmdir(path)OSError: [Errno 39] Directory not empty: '.poretools_tmp/test-data'
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="toolshed.g2.bx.psu.edu/repos/iuc/poretools_events/poretools_events (Test #2)" name="1" time="107.64825224876404">
        
            
                <error type="error" message="Tool exit code: 1"><![CDATA[
                    { "inputs": {  "input": {   "id": "eef4aa59d5bdb312",   "src": "hda"  },  "precalled": true }, "job": {  "command_line": "poretools events --pre-basecalled \u0027/galaxy/server/database/objects/a/0/2/dataset_a0212337-d50f-4699-8529-732c9b0e55dc.dat\u0027 \u003e \u0027/galaxy/server/database/objects/3/d/7/dataset_3d7af705-b289-4483-96b8-24cfa4ed21aa.dat\u0027",  "command_version": "",  "create_time": "2023-01-13T04:58:55.269625",  "dependencies": [],  "exit_code": 1,  "external_id": "gxy-pk7jr",  "galaxy_version": "21.09",  "history_id": "702fa106437bef85",  "id": "67cc327ba9dc3bf9",  "inputs": {   "input": {    "id": "eef4aa59d5bdb312",    "src": "hda",    "uuid": "a0212337-d50f-4699-8529-732c9b0e55dc"   }  },  "job_messages": [   {    "code_desc": "",    "desc": "Fatal error: Exit code 1 ()",    "error_level": 3,    "exit_code": 1,    "type": "exit_code"   }  ],  "job_metrics": [   {    "name": "runtime_seconds",    "plugin": "core",    "raw_value": "1.0000000",    "title": "Job Runtime (Wall Clock)",    "value": "1 second"   },   {    "name": "end_epoch",    "plugin": "core",    "raw_value": "1673586019.0000000",    "title": "Job End Time",    "value": "2023-01-13 05:00:19"   },   {    "name": "start_epoch",    "plugin": "core",    "raw_value": "1673586018.0000000",    "title": "Job Start Time",    "value": "2023-01-13 05:00:18"   },   {    "name": "galaxy_memory_mb",    "plugin": "core",    "raw_value": "2000.0000000",    "title": "Memory Allocated (MB)",    "value": "2000"   },   {    "name": "galaxy_slots",    "plugin": "core",    "raw_value": "1.0000000",    "title": "Cores Allocated",    "value": "1"   }  ],  "job_stderr": "Traceback (most recent call last):\n  File \"/usr/local/bin/poretools\", line 11, in \u003cmodule\u003e\n    load_entry_point(\u0027poretools==0.6.0\u0027, \u0027console_scripts\u0027, \u0027poretools\u0027)()\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 543, in main\n    args.func(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 55, in run_subtool\n    submodule.run(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/events.py\", line 13, in run\n    for fast5 in Fast5File.Fast5FileSet(args.files):\n  File \"/usr/local/lib/python2.7/site-packages/poretools/Fast5File.py\", line 110, in next\n    shutil.rmtree(PORETOOLS_TMPDIR)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 261, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 270, in rmtree\n    onerror(os.rmdir, path, sys.exc_info())\n  File \"/usr/local/lib/python2.7/shutil.py\", line 268, in rmtree\n    os.rmdir(path)\nOSError: [Errno 39] Directory not empty: \u0027.poretools_tmp/test-data\u0027\n",  "job_stdout": "",  "model_class": "Job",  "output_collections": {},  "outputs": {   "output": {    "id": "c0ba05def81d18b0",    "src": "hda",    "uuid": "3d7af705-b289-4483-96b8-24cfa4ed21aa"   }  },  "params": {   "__input_ext": "\"fast5.tar\"",   "chromInfo": "\"/cvmfs/data.galaxyproject.org/managed/len/ucsc/?.len\"",   "dbkey": "\"?\"",   "precalled": "\"true\""  },  "state": "error",  "stderr": "\nTraceback (most recent call last):\n  File \"/usr/local/bin/poretools\", line 11, in \u003cmodule\u003e\n    load_entry_point(\u0027poretools==0.6.0\u0027, \u0027console_scripts\u0027, \u0027poretools\u0027)()\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 543, in main\n    args.func(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 55, in run_subtool\n    submodule.run(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/events.py\", line 13, in run\n    for fast5 in Fast5File.Fast5FileSet(args.files):\n  File \"/usr/local/lib/python2.7/site-packages/poretools/Fast5File.py\", line 110, in next\n    shutil.rmtree(PORETOOLS_TMPDIR)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 261, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 270, in rmtree\n    onerror(os.rmdir, path, sys.exc_info())\n  File \"/usr/local/lib/python2.7/shutil.py\", line 268, in rmtree\n    os.rmdir(path)\nOSError: [Errno 39] Directory not empty: \u0027.poretools_tmp/test-data\u0027\n",  "stdout": "",  "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/poretools_events/poretools_events/0.6.1a1.1",  "tool_stderr": "",  "tool_stdout": "",  "traceback": null,  "update_time": "2023-01-13T05:00:30.763605",  "user_email": "tests@fake.org" }, "output_problems": [  "Job in error state.. tool_id: toolshed.g2.bx.psu.edu/repos/iuc/poretools_events/poretools_events/0.6.1a1.1, exit_code: 1, stderr: \nTraceback (most recent call last):\n  File \"/usr/local/bin/poretools\", line 11, in \u003cmodule\u003e\n    load_entry_point(\u0027poretools==0.6.0\u0027, \u0027console_scripts\u0027, \u0027poretools\u0027)()\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 543, in main\n    args.func(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 55, in run_subtool\n    submodule.run(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/events.py\", line 13, in run\n    for fast5 in Fast5File.Fast5FileSet(args.files):\n  File \"/usr/local/lib/python2.7/site-packages/poretools/Fast5File.py\", line 110, in next\n    shutil.rmtree(PORETOOLS_TMPDIR)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 261, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 270, in rmtree\n    onerror(os.rmdir, path, sys.exc_info())\n  File \"/usr/local/lib/python2.7/shutil.py\", line 268, in rmtree\n    os.rmdir(path)\nOSError: [Errno 39] Directory not empty: \u0027.poretools_tmp/test-data\u0027\n.",  "Job in error state.. tool_id: toolshed.g2.bx.psu.edu/repos/iuc/poretools_events/poretools_events/0.6.1a1.1, exit_code: 1, stderr: \nTraceback (most recent call last):\n  File \"/usr/local/bin/poretools\", line 11, in \u003cmodule\u003e\n    load_entry_point(\u0027poretools==0.6.0\u0027, \u0027console_scripts\u0027, \u0027poretools\u0027)()\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 543, in main\n    args.func(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py\", line 55, in run_subtool\n    submodule.run(parser, args)\n  File \"/usr/local/lib/python2.7/site-packages/poretools/events.py\", line 13, in run\n    for fast5 in Fast5File.Fast5FileSet(args.files):\n  File \"/usr/local/lib/python2.7/site-packages/poretools/Fast5File.py\", line 110, in next\n    shutil.rmtree(PORETOOLS_TMPDIR)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 261, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File \"/usr/local/lib/python2.7/shutil.py\", line 270, in rmtree\n    onerror(os.rmdir, path, sys.exc_info())\n  File \"/usr/local/lib/python2.7/shutil.py\", line 268, in rmtree\n    os.rmdir(path)\nOSError: [Errno 39] Directory not empty: \u0027.poretools_tmp/test-data\u0027\n." ], "status": "failure", "test_index": 1, "time_seconds": 107.64825224876404, "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/poretools_events/poretools_events", "tool_version": "0.6.1a1.1"}
                ]]></error>
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            Traceback (most recent call last):  File "/usr/local/bin/poretools", line 11, in <module>    load_entry_point('poretools==0.6.0', 'console_scripts', 'poretools')()  File "/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py", line 543, in main    args.func(parser, args)  File "/usr/local/lib/python2.7/site-packages/poretools/poretools_main.py", line 55, in run_subtool    submodule.run(parser, args)  File "/usr/local/lib/python2.7/site-packages/poretools/events.py", line 13, in run    for fast5 in Fast5File.Fast5FileSet(args.files):  File "/usr/local/lib/python2.7/site-packages/poretools/Fast5File.py", line 110, in next    shutil.rmtree(PORETOOLS_TMPDIR)  File "/usr/local/lib/python2.7/shutil.py", line 261, in rmtree    rmtree(fullname, ignore_errors, onerror)  File "/usr/local/lib/python2.7/shutil.py", line 270, in rmtree    onerror(os.rmdir, path, sys.exc_info())  File "/usr/local/lib/python2.7/shutil.py", line 268, in rmtree    os.rmdir(path)OSError: [Errno 39] Directory not empty: '.poretools_tmp/test-data'
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="toolshed.g2.bx.psu.edu/repos/iuc/vcfdistance/vcfdistance (Test #1)" name="0" time="124.92010021209717">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
</testsuite>